src/SDKs/EventGrid/DataPlane/Microsoft.Azure.EventGrid/Generated/Models/DeviceLifeCycleEventProperties.cs

```diff
@@ -75,22 +71,9 @@ public DeviceLifeCycleEventProperties(string deviceId = default(string), string
         [JsonProperty(PropertyName = "hubName")]
         public string HubName { get; set; }
 
-        /// <summary>
-        /// Gets or sets the event type specified for this operation by the IoT
-        /// Hub.
-        /// </summary>
-        [JsonProperty(PropertyName = "opType")]
-        public string OpType { get; set; }
```

Comment:
OK, updating the version number in the csproj to 2.0.0. Also, updating the AssemblyFileVersion to 2.0.0.0 and AssemblyVersion to 2.0.0.0. Please let me know in case that's not what you meant...

Reply:
Yep, that is what I meant, [here](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Billing/Management.Billing/Properties/AssemblyInfo.cs#L11) is an example

Reply:
Thanks, I have updated this in the latest commit.